adapter/src/test/java/com/twilio/kudu/sql/JDBCQueryIT.java

```diff
@@ -230,8 +230,8 @@ public void testSortByPrimaryKey() throws Exception {
           + "WHERE account_sid = '%s' ORDER BY account_sid, date_created, sid";
       String sql = String.format(sqlFormat, JDBCQueryIT.ACCOUNT_SID);
       String expectedPlan = "KuduToEnumerableRel\n"
-          + "  KuduProjectRel(SID=[$2], ACCOUNT_SID=[$0], DATE_CREATED=[$1])\n"
-          + "    KuduSortRel(sort0=[$0], sort1=[$1], sort2=[$2], dir0=[ASC], dir1=[ASC], dir2=[ASC], groupBySorted=[false])\n"
+          + "  KuduSortRel(sort0=[$1], sort1=[$2], sort2=[$0], dir0=[ASC], dir1=[ASC], dir2=[ASC], groupBySorted=[false])\n"
```

Comment:
This looks wrong at first but it is OK. `KuduSortRel` just sets the sorted flag and discards the `RexInputRef` of `$1`, `$2` and `$0`:

```java
  @Override
  public void implement(Implementor implementor) {
    implementor.visitChild(0, getInput());
    // create a sorted enumerator
    implementor.sorted = true;
    // set the offset
    if (offset != null) {
      final RexLiteral parsedOffset = (RexLiteral) offset;
      final Long properOffset = (Long) parsedOffset.getValue2();
      implementor.offset = properOffset;
    }
    // set the limit
    if (fetch != null) {
      final RexLiteral parsedFetch = (RexLiteral) fetch;
      final Long properFetch = (Long) parsedFetch.getValue2();
      implementor.limit = properFetch;
    }

    implementor.groupByLimited = groupBySorted;
  }
```